app/chat/welcome/page.tsx

```
@@ -0,0 +1,163 @@
+import { Suspense } from "react";
+import { auth } from "@clerk/nextjs/server";
+import ClientChatPage from "@/components/Main/ClientChatPage";
+import { ChatPageLoading } from "@/components/Main/ClientChatPage";
+import { generateId } from "@/lib/utils";
```

ESLint

[error] 'generateId' is defined but never used.

(@typescript-eslint/no-unused-vars)